resources/rendertheme.xsd

```diff
@@ -208,6 +208,14 @@
 
     </xs:complexType>
 
+    <xs:complexType name="tag-adaption">
+        <xs:attribute name="k" type="xs:string" use="required" />
+        <xs:attribute name="v" type="xs:string" use="optional" />
+        <xs:attribute name="k-osm" type="xs:string" use="required" />
+        <xs:attribute name="v-osm" type="xs:string" use="optional" />
+        <xs:attribute name="replace" type="xs:boolean" default="false" use="optional" />
```

Comment:
What is the meaning of the `replace` tag? (can be renamed to `transform`)

If create a `tag-transform` rule, isn't supposed to want `true` everywhere?

Reply:
`replace=true` means that tag is overwritten by the osm tag. Otherwise you need to lookup tag in theme class.
So when replacing is turned on, in all cases osm tag should be used. Even in render themes itself. (Thats what I meant with rethink the replacing option)

Reply:
What are the uses for not having the replace enabled, why not having it on all the time?

Reply:
It may leads to inconsistencies. As some tags are replaced and others are not (in current theme state). So it's kind of a mixture of osm and other map provider tags. I also thought about tags which arenˋt able to be transformed that simple way. But we can also give it a try with ˋreplaceˋ option only.

Reply:
Cannot understand currently the difference with existing test cases.

Would start with the simplest implementation and minimum changes, as can be tested and understood easily by all users.

If that works for everyone, then can think enhancements in the future. Could even add regular expressions in advanced step.

Reply:
What existing test cases? In this PR?

The simplest solution is to provide a list, where you can lookup tags, if you need them, without modifying the element itself. E.g. in current Building/S3DBLayer in this PR with `getKeyOrDefault()`.

But the better approach (what you mean) is to replace each tag in element (delete the old and replace by osm-tag) without need of looking up. But then you have to consider each usage of this tag in all themes (and maybe layers, too).

I suppose, I rework it and then can discuss it further :)

Reply:
> The simplest solution is to provide a list, where you can lookup tags, if you need them, without modifying the element itself.

That seems the simplest and more convenient way to handle it.
Tag transform is about using new rules in place of current ones.

> But then you have to consider each usage of this tag in all themes (and maybe layers, too).

Tag transforms should be isolated inside the theme which declares and uses them.

Reply:
What meant above is that cannot understand the need to have 2 different ways of tag transforms.

Let's start with the simplest 1st way of the lookup list and can progress from there if there is need.

Reply:
Need to clarify the purpose / workflow to understand what want to implement.
Map sources (online / offline) are created by their providers and cannot change.
Themes are created (also) by regular users, so should be as simple as possible.
Themes define tags used for rendering and can transform them to be universal?

There can never be a theme that work for all map sources, as if study the  modifications by providers, can easily see that they don't just change tags names, but also elements structure.